internal/pipe/build/build.go

```diff
@@ -134,6 +135,11 @@ func runHook(ctx *context.Context, opts builders.Options, buildEnv []string, hoo
 			return err
 		}
 
+		// shell commands need to have backslashes escaped under Windows
+		if "windows" == runtime.GOOS {
```

Comment:
```suggestion
		if runtime.GOOS == "windows" {
```

Reply:
I swapped these in this commit: `swap l-value and r-value`